woocommerce/woocommerce-ios
# Fix crash on various Settings screen

Closes: #12656 


## Description

In #12602 I removed the `Dashboard` storyboard which contains screens related to the Settings flow. This causes crashes when open Help & Support, Application Log, Privacy, and About screens from Settings.

This PR fixes this by restoring the old `Dashboard` storyboard. To avoid misunderstanding in the future, I also renamed this storyboard to `Settings`.

## Testing instructions

- Log in to the app.
- Navigate to Menu tab > Settings.
- Select Help & Support and confirm that the app doesn't crash.
- Select View Application Log and confirm that the app doesn't crash.
- Navigate back to the Settings screen and select Privacy Settings. Confirm that the app doesn't crash.
- Navigate back and select WooCommerce in the About the app section. Confirm that the app doesn't crash.

## Screenshots

About | Privacy | Help | Application Log
--- | --- | --- | ---
<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/8e7b4489-4f96-4dbd-82aa-0bbe5b198a19" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/5fcc0b81-ca64-46fa-9069-653040f91159" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/e4da68f4-774a-4408-8a5e-1d0d413d3175" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/7493d41d-ee81-42e2-8e17-3c3265d11906" width=320 />


---
